recipes/peg/all/test_package/conanfile.py

```diff
@@ -0,0 +1,8 @@
+from conans import ConanFile
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+
+    def test(self):
+        if not self.settings.os == "Windows":
```

Comment:
Why would the application not run in Windows? This test should be running also the Windows exe.

```suggestion
        if self.settings.os != "Windows":
```